src/strategies/landdao-token-tiers/examples.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "landdao-token-tiers",
+      "params": {
+        "address": "0xEa25e2b3E35c67876957EE00a28Cd912ff113F54",
+        "symbol": "LAND",
+        "tokenWeightLink": "https://ipfs.io/ipfs/QmXrDNrjX8xZu9bdNA1iiW7Ee12RADQFwwhCeLQnwqG2TJ"
```

Everything looks good :) thanks for the changes, just one last change is to accept only IPFS id here instead of API link
```suggestion
        "tokenWeightIPFS": "QmXrDNrjX8xZu9bdNA1iiW7Ee12RADQFwwhCeLQnwqG2TJ"
```